longTrip.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+import math
+
+class Trip:
+    """
+    トリップ情報を保持するクラス
+    
+    Attributes
+    ----------
+    userId: int
+        ユーザID
+    points: dict
+        キーに座標のタイムスタンプ、値にXY座標
+    
+    """
+    
+    userId = -1
+    points = {}
+    
+    def __init__(self, userId):
+        """
+        初期化
+        
+        Attributes
+        ----------
+        userId: int
+            ユーザID
+        """
+        
+        self.userId = userId
+        self.points = {}
+    
+    def addPoint(self, timeStamp, pointX, pointY):
+        """
+        座標の追加
+        
+        Attributes
+        ----------
+        timeStamp: int
+            UNIXタイムでの時刻
+        pointX: int
+            X座標
+        pointY: int
+            Y座標
+        """
+        self.points[timeStamp] = (pointX, pointY)
+        
+    def print(self):
+        print("userId: " + str(self.userId))
+        print(self.points)
+        
+    def calcDistance(self):
+        """
+        当該トリップの移動距離算出
+        pointsデータを参照し、スタート地点からゴール地点までの距離を算出
+        """
+        distance = 0
+        lastX = None
+        lastY = None
+        for point in self.points:
+            curX = self.points[point][0]
+            curY = self.points[point][1]
+            if lastX != None:
```

Comment:
PEP 8では、ある変数の値がNoneであるかどうかを調べるときには、等価性を比較する「==」や「!=」ではなく、「is」か「is not」を使用することが強く推奨されている。これはNoneはシングルトンであるために、ある値がNoneであるかどうかを調べるには同一性の比較を行うだけで十分だからである。pep8では、Noneとの比較は、"==" ではなく、isの使用が推奨されています。
(https://atmarkit.itmedia.co.jp/ait/articles/2308/01/news027.html)

```suggestion
            if lastX is not None:
```